python/src/aica_api/client.py

```diff
@@ -139,6 +141,10 @@ def set_application(self, payload: str) -> requests.Response:
         :param payload: The filepath of an application on the AICA computer, or the application content as a
         YAML-formatted string
         """
+        if payload.endswith(".yaml") and os.path.isfile(payload):
+            # FIXME print line?
```

Comment:
Fair point, adding a log output is a separate point and printing should be avoided. Let's leave it out for now.